### ISSUE TITLE
AVT 1 - Progress Indicator has DAP violations 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4243]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Progress Indicator (Default) Component
Run DAP and the following violations are present (see screenshot):
<img width="1184" alt="Screen Shot 2019-10-07 at 1 16 07 PM" src="https://user-images.githubusercontent.com/21676914/66340247-654cfd80-e90a-11e9-8a9b-6b0ccab310c3.png">

Note: Interactive Progress Indicator Component does not have the contrast issue since there are no disabled steps. Any disabled step should include aria-disabled="true".
